OrbitQt/orbitmainwindow.cpp

```diff
@@ -582,6 +582,13 @@ void OrbitMainWindow::UpdateCaptureStateDependentWidgets() {
   CaptureClient::State capture_state = app_->GetCaptureState();
   const bool is_capturing = capture_state != CaptureClient::State::kStopped;
 
+  // The detection mechanism is only implemented for the new UI, so we maintain
+  // the old behaviour for the old UI and assume the target process is always running.
+  // This can be renamed to is_target_process_running when the feature flag gets removed.
```

Comment:
IIRC we have a bug id mentioned in all other places concerning removal of the feature flag, can you add this here as well with a TODO?

Reply:
Sure. I can add that.

My reasoning for not doing that: Whenever the code involves the feature flag `FLAGS_enable_ui_beta`, it's an even stronger indication that this has to be touched when the feature flag is removed. The compiler would complain if it was forgotten.

Reply:
done.